docs/source/tutorials/index.rst

```diff
@@ -2,8 +2,9 @@ Tutorials
 =========
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 3
+
+   otx_cli/index
+   otx_sdk/index
```

Comment:
Not sure if `cli` and `sdk` fit well here. A reader might prefer to see the levels here such as `beginner`, `intermediate` and `advance`. `cli` and `sdk` could placed into either `beginner` or `intermediate`, depending on their difficulty level.

Reply:
I like the gradation `beginner` to `advanced` and I agree with your point. But what if we keep splitting according to the tool and add the difficulty division inside it? Like
- OTX CLI

> - Beginner 
>> - Train
>> - Deploy
>> - Demo

> - Advanced
>> Datumaro
>> Backbone modification

- OTX SDK
> (level division, if we'll have enough material)

Reply:
Let's omit OTE SDK part in this phase as we didn't decide yet how to refactor out it to Geti.
e.g. Move UI/X related part to SC SDK, define OTX APIs, etc...
Anyway, it will change a lot so that we'd better not to put resources for documenatation.

Reply:
Ah, so I also suggest to transpose the outline as Samet also suggested.